### DISCUSSION TITLE
Reimport stable beatmaps as hardlinks if they were previously copied across

### DISCUSSION THREAD

**Discussion by LiquidPL** [ppy/osu#24277] [Ideas]
Initially I imported my stable install to lazer some time before the hard linking was implemented, and now I'm unable to reimport them as hard links to save space without nuking the entire install (and losing scores), since as of now reimporting doesn't appear to touch the files that already exist in the database.

I think it would be a good idea for the import process to check if any of the imported files already exist in the database, and then replace them with hard links to the stable install.

**Comment by peppy** [comment]
Scores won't be lost if you delete and reimport beatmaps / skins. There are buttons in settings to do this.
